react/rules/BEL.js

```diff
@@ -0,0 +1,67 @@
+import france from '@vtex/phone/countries/BEL'
```

Comment:
Missing Changelog entry. 

There is no BEL file on: https://github.com/vtex/front.phone/tree/master/src/script/countries

Reply:
Phone validations removed since there is no BEL file in https://github.com/vtex/front.phone/tree/master/src/script/countries